build.cake

```diff
@@ -125,6 +125,7 @@ Task("Run-Unit-Tests")
                 parameters.Paths.Files.TestCoverageOutputFilePath,
                 new OpenCoverSettings {
                     ReturnTargetCodeOffset = 0,
+                    Register = "user",
```

Comment:
@devlead @gep13 That would be good even if it doesn't solve the Jenkins issue. Right now the test runners aren't working in VS so I'm running all tests via Cake when developing. Running OpenCover adds about 45 seconds to the run for me so being able to disable that would be awesome :smile: